cram2model.inputs.json

```diff
@@ -0,0 +1,55 @@
+{
+  "##_COMMENT1": "Input",
+  "Cram2TrainedModel.input_cram": "File",
+  
+  "##_COMMENT1": "Paramaters",
+  "Cram2TrainedModel.output_prefix": "String",
+  "Cram2TrainedModel.scatter_count": "Int",
+  "Cram2TrainedModel.epochs": "Int",
```

Comment:
@lucidtronix what's a good value for epochs?  I have no idea what order of magnitude even.

Reply:
100 is reasonable given this is just training from a single sample. Until we get GPUs from via cromwell though this wdl will take a long time to train a decent model.

Reply:
@ldgauthier, I'm guessing this was just a question and not a request to change the json?

Reply:
I don't think users will know what to put here if we don't provide it.  We could make it optional and default to 100.